cms/settings/base.py

```diff
@@ -979,6 +979,11 @@
     "Page",
 )
 
+# Allowed prefixes for iframe visualisations
+IFRAME_VISUALISATION_EMBED_PREFIX_LIST = env.get(
+    "IFRAME_VISUALISATION_EMBED_PREFIX_LIST", ["*.ons.gov.uk", "*.onsdigital.uk"]
+)
```

Comment:
suggestion: make this a list domain+tld. So `"ons.gov.uk,onsdigital.uk"`, then use `urllib.parse.urlparse` to extract the hostname from the provided iframe URL (`netloc` in the parsed object) and then check whether it one of the allowed domains, or part of (that is, `hostname.endswith(f".{allowed_domain})`